doc/source/getting_started/installing.rst

```diff
@@ -467,5 +467,5 @@ If that does not help, `make an issue on GitHub <https://github.com/odlgroup/odl
 .. _CVXPY: http://www.cvxpy.org/en/latest/
 .. _odlcuda: https://github.com/odlgroup/odlcuda
 .. _CUDA toolkit: https://developer.nvidia.com/cuda-toolkit
-.. _ASTRA tomography toolbox: https://github.com/astra-toolbox/astra-toolbox
+.. _ASTRA: https://github.com/astra-toolbox/astra-toolbox
```

Comment:
No, the other one is inline now, I don't see a point in 3 different targets linking to the same page.